cooker/cooker.py

```diff
@@ -179,21 +179,27 @@ def __init__(self, name, config, layers, local_conf, target, inherit):
         self.config_ = config
         self.layers_ = layers
         self.local_conf_ = local_conf
-        self.target_ = target
+        if type(target) == list:
+            self.targets_ = target
+        elif target:
+            self.targets_ = []
+            self.targets_.append(target)
+        else:
+            self.targets_ = []
         self.inherit_ = inherit
 
         self.parents_ = []  # first level parents
         self.ancestors_ = []  # all ancestors cleaned of duplicates
 
         BuildConfiguration.ALL[name] = self
 
-    def target(self):
+    def targets(self):
         if not self.buildable():
             return None
 
         for build in [self] + self.ancestors_[::-1]:
-            if build.target_:
-                return build.target_
+            if len(build.targets_) > 0:
```

Comment:
Pythonic way of checking list emptiness.

```suggestion
            if build.targets_:
```